src/IssueViz.UnitTests/Editor/QuickActions/QuickFixes/QuickFixSuggestedActionTests.cs

```diff
@@ -30,9 +34,68 @@ public class QuickFixSuggestedActionTests
         [TestMethod]
         public void DisplayName_ReturnsFixMessage()
         {
-            var testSubject = new QuickFixSuggestedAction("some fix");
+            var quickFixViz = new Mock<IQuickFixVisualization>();
+            quickFixViz.Setup(x => x.Fix.Message).Returns("some fix");
+
+            var testSubject = new QuickFixSuggestedAction(quickFixViz.Object, Mock.Of<ITextBuffer>());
+
+            testSubject.DisplayText.Should().Be("some fix");
+        }
+
+        [TestMethod]
+        public void Invoke_AppliesFixWithOneEdit()
+        {
+            var editVisualization = new Mock<IQuickFixEditVisualization>();
+            editVisualization.Setup(e => e.Edit.Text).Returns("edit");
+            
+            var quickFixViz = new Mock<IQuickFixVisualization>();            
+            quickFixViz.Setup(x => x.Fix.Message).Returns("some fix");
```

Comment:
something minor: no need to mock the fix's message as it's not related to what is being tested